bundles/pubsub/pubsub_utils/src/pubsub_serialization_provider.c

```diff
@@ -258,6 +258,21 @@ static dyn_message_type* pubsub_serializationProvider_parseDfiDescriptor(pubsub_
 //}
 //}
 
+/**
+ * Check if a pubsub serialization entry is already present (exact path)
+ *
+ * @return true if the entry is a already present
+ */
+static bool pubsub_serializationProvider_alreadyAddedEntry(pubsub_serialization_provider_t* provider, pubsub_serialization_entry_t* entry) {
```

Comment:
Probably to much work, but wouldn't it be nicer to have a map in this case?

Reply:
true, but indeed for now too much work.